libs/sdk-ui-kit/src/ResponsiveText/test/ResponsiveText.test.tsx

```diff
@@ -23,30 +24,33 @@ describe("ResponsiveText", () => {
             tagClassName: "test-class",
             title: "Responsive text title",
         };
-        const component = mount(<ResponsiveText {...props} />);
+        render(<ResponsiveText {...props} />);
 
-        expect(component.children().name()).toEqual(props.tagName);
-        expect(component.find(".test-class").exists()).toEqual(true);
-        expect(component.find(".test-class").props().title).toEqual(props.title);
+        expect(document.querySelector(".test-class")).toBeInTheDocument();
```

Comment:
Could we test the className once we select the element getByTitle like `expect(screen.getByTitle(props.title)).toHaveClass("test-class")`